client/app/components/Accordion/__tests__/Accordion.spec.jsx

```diff
@@ -1,7 +1,8 @@
 // @flow
-import { shallow } from 'enzyme';
 import React from 'react';
 import { Accordion } from '../index';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Is this necessary? If so I think we can place [/client/app/libs/testHelper.js](https://github.com/ifmeorg/ifme/blob/master/client/app/libs/testHelper.js) so that it's globally available.

Reply:
Do you mean line 5? Because when I removed it from the spec file and tried to put it inside testHelper I get an error when running the tests (even after using another defineProperty for expect):

![image](https://user-images.githubusercontent.com/25817917/84674753-a8488180-af2b-11ea-8422-efde447db9a0.jpeg)

Reply:
Interesting, what happens if you import `import '@testing-library/jest-dom'` instead? Does that work?